contracts/starknet/src/tests/test_escrow_claim.cairo

```diff
@@ -0,0 +1,339 @@
+mod Escrow {
+    use core::array::ArrayTrait;
+    use core::to_byte_array::FormatAsByteArray;
+    use core::serde::Serde;
+    use core::traits::Into;
+    use starknet::{EthAddress, ContractAddress};
+    use integer::BoundedInt;
+
+    use snforge_std::{declare, ContractClassTrait, L1Handler, L1HandlerTrait};
+    use snforge_std::{CheatTarget, start_prank, stop_prank, start_warp, stop_warp};
+
+    use yab::mocks::mock_Escrow_changed_functions::{IEscrow_mock_changed_functionsDispatcher, IEscrow_mock_changed_functionsDispatcherTrait};
+    use yab::mocks::mock_pausableEscrow::{IEscrow_mockPausableDispatcher, IEscrow_mockPausableDispatcherTrait};
+    use yab::interfaces::IERC20::{IERC20Dispatcher, IERC20DispatcherTrait};
+    use yab::escrow::{IEscrowDispatcher, IEscrowDispatcherTrait, Order};
+    use yab::interfaces::IEVMFactsRegistry::{
+        IEVMFactsRegistryDispatcher, IEVMFactsRegistryDispatcherTrait
+    };
+
+    use yab::tests::utils::{
+        constants::EscrowConstants::{
+            USER, OWNER, MM_STARKNET, MM_ETHEREUM, ETH_TRANSFER_CONTRACT, ETH_USER, ETH_USER_2, ETH_USER_3
+        },
+    };
+
+    use openzeppelin::{
+        upgrades::{
+            UpgradeableComponent,
+            interface::{IUpgradeable, IUpgradeableDispatcher, IUpgradeableDispatcherTrait}
+        },
+    };
+
+    fn setup() -> (IEscrowDispatcher, IERC20Dispatcher) {
+        setup_general(BoundedInt::max(), BoundedInt::max())
+    }
+
+    // fn setup_approved(approved: u256) -> (IEscrowDispatcher, IERC20Dispatcher){
```

Comment:
why is it commented

Reply:
removed on e76e5e4abfc0c95729731138ee18df718cceda40